catalog-info.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "-mo-webapp"
```

Think this needs changed 👀

```suggestion
  name: "rpx-xui-mo-webapp"
```